.github/dependabot.yml

```diff
@@ -9,6 +9,8 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+    - dependency-name: org.codehaus.groovy:groovy
```

Comment:
Maybe we will also need to ignore `org.codehaus.groovy:groovy:sources`? I'm not sure.

Reply:
I guess we can add later if it keeps popping up